download.sh

```diff
@@ -56,3 +57,5 @@ download_if_not_exists https://github.com/yanghua/incubator-hudi/releases/downlo
 
 LOKI_APPENDER_JAR_NAME=log4j2-appender-nodep
 download_if_not_exists ${MAVEN_MIRROR}/pl/tkowalcz/tjahzi/${LOKI_APPENDER_JAR_NAME}/${LOKI_APPENDER_VERSION}/${LOKI_APPENDER_JAR_NAME}-${LOKI_APPENDER_VERSION}.jar
+
+download_if_not_exists ${MAVEN_MIRROR}/org/slf4j/jcl-over-slf4j/1.7.36/jcl-over-slf4j-1.7.36.jar 
```

Comment:
@pan3793 Do we need to extract the version number `1.7.36`?

Reply:
slf4j's compatible is pretty good, hardcode is fine